front/app/components/FormBuilder/components/FormBuilderSettings/ContentSettings/index.tsx

```diff
@@ -123,22 +123,31 @@ export const ContentSettings = ({
             buttonStyle="secondary"
             onClick={onClose}
             minWidth="160px"
+            width={
+              !isNilOrError(field.isDeleteEnabled) && !field.isDeleteEnabled
```

Comment:
Since `isDeleteEnabled` is an optional boolean, I think we can just do remove the isNil` check and use something like `!field.isDeleteEnabled ? ...`

We can apply something similar on line 134